addons/auth_signup/views/res_users_views.xml

```diff
@@ -38,12 +38,9 @@
             <field name="model">res.users</field>
             <field name="inherit_id" ref="base.view_users_tree"/>
             <field name="arch" type="xml">
-                <xpath expr="//tree" position="attributes">
-                    <attribute name="decoration-info">state == 'new'</attribute>
-                </xpath>
-                <xpath expr="//field[@name='login_date']" position="after">
-                    <field name="state" invisible="1"/>
-                </xpath>
+                <tree>
```

Comment:
That's strange as you inherit from a view but rewrite its tree ? Either you want an independent view, either you modify existing view with xpath, but I don't really understand this override. What is its purpose ?

Reply:
True, after having a look at this it seems that using xpath is better.